src/taipy/core/exceptions/exceptions.py

```diff
@@ -43,7 +43,7 @@ class NoData(Exception):
 
 
 class UnknownDatabaseEngine(Exception):
-    """Raised if the database engine is not known when creating a connection with a SQLDataNode."""
+    """Raised if the database engine is not known when creating a connection with an SQLDataNode."""
```

Comment:
I noticed that we have some inconsistency of "a SQL..." or "an SQL..." throughout not just taipy-core, but also in taipy-doc (I haven't checked other repos). Please fix it while you're on the topic. Thanks!